### PR TITLE
Fix analytics try catch

### PR DESCRIPTION
**Pull request by andrepimenta**
**Description**

Fixes try catch on analytics log.

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

**Review by rickycodes — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Current Develop Branch
```
Comment (estebanmino) [inline CHANGELOG.md:3]: the version is missing here

Comment (estebanmino) [reply CHANGELOG.md:3]: ## v2.2.0 - Apr 21 2021
 maybe

Comment (andrepimenta) [reply CHANGELOG.md:3]: Thanks!!

**Review by estebanmino — APPROVED** [review]
🚀

**Pull request merged**